src/start.ts

```diff
@@ -67,11 +68,16 @@ export const handler = async (_event: any = {}): Promise<any> => {
           nodeSettings.airnodeWalletMnemonic
         );
 
-        let currentBlock: number;
-        try {
-          currentBlock = await provider.getBlockNumber();
-        } catch (error) {
-          printError(error);
+        // Fetch current block number from chain via provider
+        const retryableGetBlockNumberCall = node.utils.retryOnTimeout(
+          DEFAULT_RETRY_TIMEOUT_MS,
+          () => provider.getBlockNumber()
```

Comment:
Echoing comments I received on my own PR using this library: you can inline this function